### ISSUE TITLE
BaseUrl in App section of nuxt.config.ts and manifest.webmanifest

### ISSUE THREAD

**Issue by GYSRichardGUION** [vite-pwa/nuxt#46]
I have a Nuxt 3 project using Vite-Pwa-Nuxt plugin.
When I deploy the application on the domain root without a particular path, so with an empty baseUrl of the app (app section in nuxt.config.ts), the manifest.webmanifest file is generated in the public folder and it works correctly.

When I deploy the application on a  path of the domain, for example myPath, so with baseUrl equal myPath, the manifest.webmanifest file is generated in the myPath subdirectory of the public folder and it does not work. The manifest is not found by the navigator.

What am I doing wrong ?

**Comment by userquin** [comment]
@GYSRichardGUION so, are you using `app.baseURL` in nuxt config? we need it at build time.

**Comment by userquin** [comment]
it seems these lines (https://github.com/vite-pwa/nuxt/blob/main/src/module.ts#L67-L71) writting webmanifest using baseURL as folder (removing `baseURL` seems to fix it):
```ts
  const manifestDir = join(nuxt.options.buildDir, 'manifests')
    nuxt.options.nitro.publicAssets = nuxt.options.nitro.publicAssets || []
    nuxt.options.nitro.publicAssets.push({
      dir: manifestDir,
      baseURL: nuxt.options.app.baseURL,
      maxAge: 0,
    })
```

Running `nr dev:preview:build` after removing `baseURL` and building the plugin works: cannot test `nr dev:preview:generate` since we cannot change baseURL.

I'll try to fix dev server  and `workbox-***.js` assets.

/cc @danielroe

**Comment by userquin** [comment]
ok, found the problem in `vite-plugin-pwa` DevPlugin, we need to remove base if present or just don't normalize the id:

![imagen](https://github.com/vite-pwa/nuxt/assets/6311119/b346f2a9-e057-4320-9a45-cf4f941b0fe0)

**Comment by userquin** [comment]
@GYSRichardGUION should be fixed in `v0.0.10`

**Comment by GYSRichardGUION** [comment]
Thanks @userquin. It works correctly now
